### ISSUE TITLE
Replace project logo

### ISSUE THREAD

**Issue by Danimal696** [OpenDungeons/OpenDungeons#1052]
Dakal made an awesome replacement for our logo, i vote for it to replace the current one.

Lookit here:
http://forum.freegamedev.net/viewtopic.php?f=38&t=6703

**Comment by Bertram25** [comment]
@OpenDungeons/developers @OpenDungeons/artists  Please, let's do it. The new logo is so nice! :D

**Comment by akien-mga** [comment]
I like it too, and it should render fine at small scale :)

**Comment by Bertram25** [comment]
I'll work on that one after the current PR. :)

**Comment by Bertram25** [comment]
@Danimal696 @akien-mga @hwoarangmy  I'd like to tackle that one next to the map creation PR. I can easily replace the icon and the demonic part of the logo. But what about the other letters?
WDYT about keeping the same style and adding some painterly effect on them?
(I know it's difficult to imagine like that, I'll upload something once I'm more or less happy about it.)

**Comment by hwoarangmy** [comment]
> WDYT about keeping the same style and adding some painterly effect on them?

I would see it depends on what you can achieve ;-)

**Comment by Danimal696** [comment]
Ok, ill wait to see it as well if its a good effect, otherwise no problem

**Comment by Bertram25** [comment]
I tried something but it went bad in the end:
![odlogo](https://cloud.githubusercontent.com/assets/765917/12250233/b8a81eac-b8c5-11e5-9a42-e9f2bfebbc86.png)
I'll have a new go at editing the letters to make their borders feel like the logo but you have material to laugh at me now. ;) The only thing I'm fine with, is to make the 'n' partially above the logo.
Feel free to comment in any case.

**Comment by MCMic** [comment]
Feels hard to read. Unless you know it, you won’t read «Dungeons» the first time, having a round thing for a D is counter-intuitive.

**Comment by alketii** [comment]
It looks good but it might be controversial.

**Comment by Bertram25** [comment]
> Feels hard to read. Unless you know it, you won’t read «Dungeons» the first time, having a round thing for a D is counter-intuitive.

Yeah, I feel the same. I don't have an actual solution to this yet.

> It looks good but it might be controversial.

In what way?

**Comment by alketii** [comment]
@Bertram25 when I think about it now, it looks like I didn't think it through xD (didn't have the morning coffee:P)
But overall I think logo should be improved a bit, to have more "solid" colors, instead of looking like a painting.

**Comment by Bertram25** [comment]
> But overall I think logo should be improved a bit, to have more "solid" colors, instead of looking like a painting. 

Ah well, the essence of the "new" logo was to look like being painted. I'm also sure the artist started from something more "solid" and applied painting effects on it.

I could replace the "O", but then we would have the "old" D still visible. I could also replace the whole writing, but I need to find a decent replacement.

**Comment by hwoarangmy** [comment]
> having a round thing for a D is counter-intuitive.

Well, first time I saw the logo, I didn't even see the O ^^
I guess it might be good to try to have have the logo on the left with:
pen
ungeons

**Comment by Bertram25** [comment]
Ah, forgot to answer, but yes, your idea looks good. :) I'll give it a try.

**Comment by akien-mga** [comment]
> I guess it might be good to try to have have the logo on the left with:
> pen
> ungeons

I think it will be even harder to read :/ If it's not readable in Open<logo>ungeons, it won't be readable in the above proposal either. The D is only readable when you know it's there IMO :)
If we really want to use the logo in the name, we could simply go with <logo>penDungeons IMO.

**Comment by Bertram25** [comment]
Hmm, ok, I'll try replacing only the O, and see how I can deal with the D.

**Comment by hwoarangmy** [comment]
> If we really want to use the logo in the name, we could simply go with <logo>penDungeons IMO.

Not stupid ^^

**Comment by Bertram25** [comment]
Just to say that Lilou is also on this one. :>
